djimporter/importers.py

```diff
@@ -130,10 +130,18 @@ def is_valid(self):
         if self.errors:
             return False
 
+        num_lines = len(self.csv_file) - 1
+        # Status progress will be saved 10 times
+        block_lines = int(num_lines / 10)
         for line_number, line in enumerate(self.csv_reader, start=2):
             # line is a dictionary with the fields of csv head as key
             # and values of the row as value of the dictionary
             self.process_line(line, line_number)
+            if log is not None and (line_number - 1) % block_lines == 0:
+                progress = (line_number - 1) * 100 / num_lines
+                log.progress = round(progress)
+                log.num_rows = line_number - 1
```

Comment:
What does represent `line_numer - 1`? (as is calculated 3 times, maybe could be stored on a variable with a descriptive name).

Reply:
line_number - 1 is the current processed row (header is not a valid row). Stored in a variable called row.